integration_tests/src/regular_nft_tests.rs

```diff
@@ -247,6 +247,59 @@ async fn test_search_by_owner_with_show_zero_balance() {
 
     let request: SearchAssets = serde_json::from_str(request).unwrap();
     let response = setup.das_api.search_assets(request, mutexed_tasks.clone()).await.unwrap();
+
+    assert_eq!(response["items"].as_array().unwrap().len(), 2);
+    insta::assert_json_snapshot!(name, response);
+}
+
+#[tokio::test]
+#[serial]
+#[named]
+async fn test_search_by_owner_with_show_zero_balance_false() {
+    let name = trim_test_name(function_name!());
+    let setup = TestSetup::new_with_options(
+        name.clone(),
+        TestSetupOptions { network: Some(Network::Mainnet), clear_db: true },
+    )
+        .await;
+
+    let seeds: Vec<SeedEvent> = seed_token_mints([
+        "HxhWkVpk5NS4Ltg5nij2G671CKXFRKPK8vy271Ub4uEK", // mint for fungible acc
+    ]);
+    index_seed_events(&setup, seeds.iter().collect_vec()).await;
+
+    let seeds: Vec<SeedEvent> = seed_accounts([
```

Comment:
Call `index_seed_events()` for these seeds with empty token acc and fungible token before seeds with mint for fungible acc, which is above, and it will break the test.
And also you will receive condition described in ticket.